### ISSUE TITLE
BG-706 (Techniques); Description; Missing Checkbox - Grouped notices - DPS and framework agreement

### ISSUE THREAD

**Issue by AlesMMR** [eForms/eForms#130]
In art. 50 of classsic directive is stated that buyer can send grouped notices within 30 days of the end of each quarter in case DPS and framework agreement (FA). 
Buyers use the same form for annoucment of FA concluded or DPS and for annoucment of contracts based on FA or DPS - F03. There is an info, that buyer is announicng conclusion of FA or DPS but there is no box to state that publication concern annoucement of contract based on FA or DPS. 

In other words we would like to add checkbox "This notice involves information on award contract based on DPS or FA". It will help us to indetify such notices in statistics.

**Comment by JachymHercher** [comment]
Great. Concerning the checkbox, please note that that's rather a presentation feature - the data can be stored in a codelist, but to users (whether those filling in the form or those reading the published notice) it can still be displayed as a checkbox or a series of checkboxes. Every implementer can make this choice himself, depending on what he thinks will be the most understandable for his users.
